### PR TITLE
Es 1409

### PR DESCRIPTION
**Pull request by wrighbr**

### CODE REVIEW

File: template.yaml [hunk 1]
```diff
@@ -40,6 +40,25 @@ spec:
...
               - Group
+    - title: Choose deployment options
+      required:
+        - use_datadog_apm
+        - deployment_type
+      properties:
+        use_datadog_apm:
+          title: Enable datadog apm?
+          type: string
+          enum:
```
Comment (Gaardsholt) [inline template.yaml:51]: Shouldn't we us a boolean instead of a string?

File: template/docs/swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+basePath: /
+host: test-bw1o-go.k8s.bestcorp.net
```
Comment (Gaardsholt) [inline template/docs/swagger.yaml:2]: Should it have this hardcoded values?

**Review by Gaardsholt — APPROVED** [review]

File: template/docs/swagger.yaml (hunk 2 above)
Comment (wrighbr) [reply template/docs/swagger.yaml]: Added cooker cutter var instead.

File: template.yaml (hunk 1 above)
Comment (wrighbr) [reply template.yaml]: convert to bool. much better solution then copy and paste from example file :)

**Pull request merged**